README.md

```diff
@@ -1,208 +1,262 @@
+
+## Limitation
+
+This project assumes that the mobile app and the "back-end" part will be co-located in the same application.
+
+At this moment this project does not support:
+- Off-line hybrid mobile
+- Anonymous access
+- Multiple devices per user
 
 ## Contributing
 
-For more information on contributing to this repository visit [Contributing to a GitHub repository](https://world.mendix.com/display/howto50/Contributing+to+a+GitHub+repository)!
- 
-## Description
-
-This project gives you all the necessary widgets, javascript, java and modules necessary for you to both send and receive push notifications. In order to find information on how to build your Mendix apps into phonegap applications please refer to this documentation: [Mendix mobile] (https://world.mendix.com/display/refguide5/Mobile)
-
-## Mendix setup
-In order for the push notifications to work in Mendix you must have a few things set up.
-- Import module into a 5.14.1 or higher project.
-- The microflow AfterStartup_PushNotifications must be included in your after startup flow
-- The index.html and components.json must include a reference to a jquery library.
-`<script type="text/javascript" src="js/jquerymin.js"></script>`
-- The index.html and the components.json file must include the following reference to this javascript library
-`<script type="text/javascript" src="widgets/pushNotifications/lib/PushNotification.js"></script>`
-- The push notification snippet must be included on all layouts for mobile and tablet.
-- The pages AppleAdministration, GoogleAdministration and Device_Overview must be connected up to the navigation.
-- You must set up the [apple server](#setting-up-apple-push-notification-server) and [google server](#setting-up-google-cloud-messaging-server) using the documentation bellow
-- The phonegap push plugin must be included in the config.xml, more information can be found [here](#creating-phonegap-app)
-- The widget is connected up to the google settings object. 
-- Encryption module from the appstore must be added to the project.
-
-The application included in the test project can be used as reference.
-
-## Components.json
-All mendix applications that want to utilise phonegap require a components file. The components file describes all the javascript and css files that need to be loaded when the application is loaded on the device. Bellow is an example of the components file that would be required for this module.
+For more information on contributing to this repository visit [Contributing to a GitHub repository](https://world.mendix.com/display/howto6/Contributing+to+a+GitHub+repository)!
+
+## Implementation Guide
+
+### Step 1 - Create an mpk of the PushNotifications module
+
+We need to extract the module from this project before starting with the implementation. Walk through these following steps:
```

Comment:
Was suggesting we could do 'Walk through the following steps' ... but pretty minor detail, either is fine I think